shared/constants/security-provider.ts

```diff
@@ -55,10 +55,11 @@ export enum BlockaidResultType {
   Malicious = 'Malicious',
   Warning = 'Warning',
   Benign = 'Benign',
-
   // MetaMask defined result types
   Failed = 'Failed',
   NotApplicable = 'NotApplicable',
+  /* eslint-disable-next-line @typescript-eslint/no-shadow */
+  Error = 'Error',
```

Comment:
when we resolve the merge conflict it should bring in this change
```suggestion
  Errored = 'Error',
```
removing the eslint disable line

will need to update the other instances in this PR with it